### PR TITLE
CTW-692: part 1 - med statement

### PR DESCRIPTION
**Pull request by sarahmangels**

### CODE REVIEW

File: src/services/fqs/queries/fragments.ts [hunk 1]
```diff
@@ -94,3 +94,24 @@ export const fragmentPatient = gql`
     }
   }
 `;
+
+export const fragmentReference = gql`
```
Comment (sarahmangels) [inline src/services/fqs/queries/fragments.ts:98]: see TODOs in here --- not sure how deep we need to go on References. They just keep going 😱

File: src/fhir/models/medication-statement.ts [hunk 2]
```diff
@@ -27,6 +27,18 @@ export class MedicationStatementModel extends FHIRModel<fhir4.MedicationStatemen
 
   readonly builderPatientRxNormStatus?: Record<string, string>;
 
+  get aggregatedFrom(): Reference[] {
```
Comment (sarahmangels) [inline src/fhir/models/medication-statement.ts:30]: just alphabetizing - no changes

File: src/fhir/medication.ts [hunk 3]
```diff
@@ -69,7 +69,7 @@ export function getIdentifyingRxNormCoding(
   const rxNorm = codeableConcept?.coding?.find(
     (code) =>
       // must be an RxNorm code
-      code.system === SYSTEM_RXNORM && code.extension === undefined
+      code.system === SYSTEM_RXNORM && !code.extension?.some((e) => e.url === SYSTEM_ENRICHMENT)
```
Comment (sarahmangels) [inline src/fhir/medication.ts:72]: switched this because the FQS data has code.extension as `null` so the existing check didnt work

**Review by patrickvalle — APPROVED** [review]
lgtm

**Pull request merged**